search-solr/src/main/java/edu/unc/lib/boxc/search/solr/services/SearchStateFactory.java

```diff
@@ -272,6 +272,12 @@ private void populateQueryableFields(SearchState searchState, Map<String,String[
             }
             if (searchSettings.searchableFields.contains(key)) {
                 searchFields.put(key, value);
+            } else if (SearchSettings.FIELDS_RANGE_SEARCHABLE.contains(key)) {
+                try {
+                    rangeFields.put(key, new SearchState.RangePair(value));
+                } catch (ArrayIndexOutOfBoundsException e) {
```

Comment:
```suggestion
                } catch (ArrayIndexOutOfBoundsException | IllegalArgumentException e) {
```
Unless it would be better for the exception to be communicated back to the user, but i think adding the catch here would be inline with the existing behavior of catching ArrayIndexOutOfBoundsException